docs/MO_DG/prepare_model/convert_model/tf_specific/Convert_XLNet_From_Tensorflow.md

```diff
@@ -59,36 +59,36 @@ xlnet_config = xlnet.XLNetConfig(json_path=XLNET_CONFIG_PATH)
 run_config = xlnet.RunConfig(is_training=False, use_tpu=False, use_bfloat16=False, dropout=0.1, dropatt=0.1,)
 
 
-sentence_features_input_idx = tf.placeholder(tf.int32, shape=[LENGTHS, BATCH], name='input_ids')
-sentence_features_segment_ids = tf.placeholder(tf.int32, shape=[LENGTHS, BATCH], name='seg_ids')
-sentence_features_input_mask = tf.placeholder(tf.float32, shape=[LENGTHS, BATCH], name='input_mask')
+sentence_features_input_idx = tf.compat.v1.placeholder(tf.int32, shape=[LENGTHS, BATCH], name='input_ids')
```

Comment:
we use xlnet module here. does it work with tf2?

Reply:
From this page https://github.com/zihangdai/xlnet

> As of June 19, 2019, this code base has been tested with TensorFlow 1.13.1 under Python2.

So looks like no.

Reply:
Did you check that the code you've changed work for both tf1.x and tf2.x? I don't have enough confidence without your confirmation.

Reply:
Do we really need this check? If yes, then we need such checks for all other model modules/repositories and do checks for all future dependency changes.

Maybe is is better to have some message about Python/TensorFlow versions used in model repo?

Reply:
Note about original repository code added.